[2024-12-02T14:43:01.140+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-02T14:43:01.173+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: consume_kafka_to_hive.consume_kafka_messages manual__2024-12-02T14:42:54.104645+00:00 [queued]>
[2024-12-02T14:43:01.199+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: consume_kafka_to_hive.consume_kafka_messages manual__2024-12-02T14:42:54.104645+00:00 [queued]>
[2024-12-02T14:43:01.201+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-02T14:43:01.272+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): consume_kafka_messages> on 2024-12-02 14:42:54.104645+00:00
[2024-12-02T14:43:01.279+0000] {standard_task_runner.py:72} INFO - Started process 2031 to run task
[2024-12-02T14:43:01.283+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'consume_kafka_to_hive', 'consume_kafka_messages', 'manual__2024-12-02T14:42:54.104645+00:00', '--job-id', '144', '--raw', '--subdir', 'DAGS_FOLDER/consume_kafka_to_hive.py', '--cfg-path', '/tmp/tmpvck87afw']
[2024-12-02T14:43:01.286+0000] {standard_task_runner.py:105} INFO - Job 144: Subtask consume_kafka_messages
[2024-12-02T14:43:01.444+0000] {task_command.py:467} INFO - Running <TaskInstance: consume_kafka_to_hive.consume_kafka_messages manual__2024-12-02T14:42:54.104645+00:00 [running]> on host 45770b1a429e
[2024-12-02T14:43:01.711+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='consume_kafka_to_hive' AIRFLOW_CTX_TASK_ID='consume_kafka_messages' AIRFLOW_CTX_EXECUTION_DATE='2024-12-02T14:42:54.104645+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-02T14:42:54.104645+00:00'
[2024-12-02T14:43:01.713+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-02T14:43:02.126+0000] {hive.py:475} INFO - USE `default`
[2024-12-02T14:43:02.671+0000] {hive.py:475} INFO - 
    CREATE TABLE IF NOT EXISTS sales (
        id STRING,
        invoice_id STRING,
        branch STRING,
        city STRING,
        customer_type STRING,
        gender STRING,
        product_line STRING,
        unit_price FLOAT,
        quantity INT,
        tax_5_percent FLOAT,
        total FLOAT,
        `date` STRING,
        time STRING,
        payment STRING,
        cogs FLOAT,
        gross_margin_percentage FLOAT,
        gross_income FLOAT,
        rating FLOAT
    ) STORED AS ORC
    
[2024-12-02T14:43:02.735+0000] {consume_kafka_to_hive.py:63} INFO - Subscribed to topic 'topic-hive'
[2024-12-02T14:51:25.391+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-12-02T14:51:25.392+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-12-02T14:51:25.394+0000] {process_utils.py:132} INFO - Sending 15 to group 2031. PIDs of all processes in the group: [2031]
[2024-12-02T14:51:25.394+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 2031
[2024-12-02T14:51:25.840+0000] {taskinstance.py:3093} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-12-02T14:51:25.910+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-02T14:51:25.930+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=2031, status='terminated', exitcode=0, started='14:43:00') (2031) terminated with exit code 0
